metro.config.js

```diff
@@ -1,4 +1,9 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable functional/immutable-data */
```

Comment:
Are these changes required?

Reply:
Our lint is actually firing a lot of errors inside metro.config due to our strict rules, to strenghten the readability and avoid all the red underlines atm I guess we can leave this comments